plugin/pkg/scheduler/api/validation/validation.go

```diff
@@ -36,7 +36,7 @@ func ValidatePolicy(policy schedulerapi.Policy) error {
 
 	binders := 0
 	for _, extender := range policy.ExtenderConfigs {
-		if extender.Weight <= 0 {
+		if extender.PrioritizeVerb != "" && extender.Weight <= 0 {
```

Comment:
nit and optional: s/extender.PrioritizeVerb != ""/len(extender.PrioritizeVerb) > 0/

Reply:
Done.